### PR TITLE
Including changes to nrcan branch

### PR DESCRIPTION
**Pull request by ckirney**
-commented out require statements for libraries that caused problems when bundled with OpenStudio
-include scaling for shw consumption
-add VSD chiller ecm
-add option to put individual residential spaces on individual air loops, or all in one air loop
-add updates for NECB 2020
-other fixes/new ecms

### CODE REVIEW

File: lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb [hunk 1]
```diff
@@ -1643,8 +1643,10 @@ def setup_chw_loop_with_components(model, chw_loop, chiller_type)
     sizing_plant.setDesignLoopExitTemperature(7.0)
     sizing_plant.setLoopDesignTemperatureDifference(6.0)
 
-    # pump = OpenStudio::Model::PumpConstantSpeed.new(model)
-    chw_pump = OpenStudio::Model::PumpConstantSpeed.new(model)
+    # Note: pump of 'chilled water loop' has been changed to the variable one as the constant one caused fatal errors for LargeOffice-Yellowknife-NaturalGas for some ECMs and inputs.
+    # Fatal error was: 'CheckForRunawayPlantTemps: Simulation terminated because of run away plant temperatures, too cold' OR '..., too hot' for the PlantLoop of 'Chilled Water Loop'.
```
Comment (mdahlhausen) [inline lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb:1647]: This indicative of a control issue, and just switching to pump variable speed may mask the problem.

File: lib/openstudio-standards/standards/necb/NECB2020/necb_2020.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # This class holds methods that apply NECB2020 rules.
 
 # Notes for adding new version of NECB:
-#  Essentially all you need to do is copy this file to a new folder and update the class name. Only add json files and other rb
-#  files if the content has changed. Do not forget to update the class name in the rb files.
+#  Essentially all you need to do is copy this file to a new folder and update the class name (only the initialize and load_standards_database_new methods ae required,
```
Comment (mdahlhausen) [inline lib/openstudio-standards/standards/necb/NECB2020/necb_2020.rb:4]: spelling error

File: lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb [hunk 3]
```diff
@@ -1034,16 +1123,40 @@ def apply_efficiency_ecm_hs09_ccashp_baseboard(model)
               supply_fan = fan # assume supply fan has higher pressure drop
             end
           end
-          fan_power = supply_fan.autosizedMaximumFlowRate.to_f * max_pd / supply_fan.fanTotalEfficiency.to_f
+          # There is an error in EnergyPlus in the estimated capacity of the coil "CoilCoolingDXVariableSpeed".
```
Comment (mdahlhausen) [inline lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb:1126]: what is the EnergyPlus error?

**Review by mdahlhausen — CHANGES_REQUESTED** [review]
Some minor comments and clarifications

File: lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb (hunk 3 above)
Comment (khaddad) [reply lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb:1126]: @mdahlhausen This error was confirmed with the EnergyPlus team and they actually have a ticket for it. The reported autosized capacity of this coil is not correct. In this code a work-around is implemented to correct the capacity till a a fix is implemented by the E+ team.

Comment (mdahlhausen) [reply lib/openstudio-standards/standards/necb/ECMS/hvac_systems.rb:1126]: got it thanks

File: lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb (hunk 1 above)
Comment (srgilani) [reply lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb:1647]: I also tried to solve this issue through availability manager (AvailabilityManagerLowTemperatureTurnOff and AvailabilityManagerHighTemperatureTurnOff) without changing the pump type, however it did not work. We will investigate it further in the future.

File: lib/openstudio-standards/standards/necb/NECB2020/necb_2020.rb (hunk 2 above)
Comment (ckirney) [reply lib/openstudio-standards/standards/necb/NECB2020/necb_2020.rb]: Fixed.

File: lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb (hunk 1 above)
Comment (mdahlhausen) [reply lib/openstudio-standards/standards/necb/NECB2011/hvac_systems.rb:1647]: ok.  @ckirney @srgilani I'll go ahead and merge but this is something you will want to address.

**Review by mdahlhausen — APPROVED** [review]

**Pull request merged**